src/main/java/com/amazonaws/services/sqs/AmazonSQSRequesterClientBuilder.java

```diff
@@ -87,7 +88,20 @@ public AmazonSQSRequesterClientBuilder withIdleQueueSweepingPeriod(int period, T
         setIdleQueueSweepingPeriod(period, timeUnit);
         return this;
     }
-    
+
+    public long getIdleQueueHeartbeatInterval() {
+		return idleQueueHeartbeatInterval;
```

Comment:
No tabs please! :) Weird that it's only this file affected AFAICT.

Reply:
Weird that only this file was indented with tabs, should be fixed now.